api_v3/services/MediaService.php

```diff
@@ -149,12 +149,20 @@ protected function replaceResource(KalturaResource $resource, entry $dbEntry, $c
 		}
 		else
 		{
-
 			$tempMediaEntry = new KalturaMediaEntry();
-		 	$tempMediaEntry->type = $dbEntry->getType();
+			$tempMediaEntry->type = $dbEntry->getType();
 			$tempMediaEntry->mediaType = $dbEntry->getMediaType();
-			$tempMediaEntry->conversionProfileId = $dbEntry->getConversionQuality();
 
+			if ( !$conversionProfileId ) {
+				$originalConversionProfileId = $dbEntry->getConversionQuality();
+				$conversionProfile = conversionProfile2Peer::retrieveByPK($originalConversionProfileId);
+				if ( is_null($conversionProfile) || $conversionProfile->getType() == ConversionProfileType::LIVE_STREAM )
```

Comment:
Instead of checking == Live, check != Media, just in case we will have another type in the future.